src/client/automation/playback/click/index.js

```diff
@@ -225,12 +225,20 @@ export default class ClickAutomation {
         return cursor
             .buttonUp()
             .then(() => {
-                var prevElementForEvent = this.eventArgs.element;
-
-                this.eventArgs            = this._calculateEventArguments();
-                this.eventState.skipClick = prevElementForEvent !== this.eventArgs.element;
-
-                eventSimulator.mouseup(this.eventArgs.element, this.eventArgs.options);
+                var topElementEventArgs = this._calculateEventArguments();
```

Comment:
Before we assing ` this.eventArgs` before raise `mouseup`. And raise `click` for `this.eventArgs.element`.
Should not we do it?